content/info/call-participation/review-instructions.md

```diff
@@ -6,74 +6,189 @@ contact: papers@ieeevis.org
 sidebar: call-for-participation
 ---
 
-IEEE VIS 2020 employs a two-phase review process ...
+The success of the IEEE VIS 2020 papers program relies on a knowledgeable, experienced, and ethical community of reviewers. IEEE VIS 2020 employs a two-phase, [(author-optional) double-blind review](#http://ieeevis.org/year/2020/info/call-participation/paper-submission-guidelines#important-submission-requirements) process that follows the review process for IEEE TVCG, the journal in which most IEEE VIS full papers are published. All papers are reviewed by at least 4 reviewers, at least 2 of which are IPC members and at least 2 of which are external members of the community. All reviewers are bound to the [ethics guidelines of the IEEE VGTC](#http://vgtc.org/about-us/conferences/ethics-guidelines).
 
-## Review Assignment Phase
+This page provides detailed instructions for both IPC reviewers and external reviewers.
 
-The paper submission deadline is March 31st. After paper submissions, the paper chairs assign primary and secondary reviewers, with an internal deadline of April 13th for assignments. 
+[IPC Reviewing Instructions](#IPC Reviewing Instructions)
```

Comment:
The link to the headers is all-lowercase with underscore (`_`) characters in the spaces; so this line would become `#ipc_reviewing_instructions`).  I'm happy to make these fixes if you'd like.